packages/react-components/react-dialog/src/components/DialogSurface/useDialogSurfaceStyles.styles.ts

```diff
@@ -58,6 +58,12 @@ const useRootResetStyles = makeResetStyles({
   [MEDIA_QUERY_BREAKPOINT_SELECTOR]: {
     maxWidth: '100vw',
   },
+
+  opacity: 0,
+  boxShadow: '0px 0px 0px 0px rgba(0, 0, 0, 0.1)',
```

Comment:
Will the hardcoded color work as expected in all light, dark and HC?

Reply:
For the `boxShadow`?! yes, it works fine, as this is the initial state before the animation, and the final one is `tokens.shadow64`